RELEASE.md

```diff
@@ -0,0 +1,127 @@
+# Release a new version
+
+1) Update add-on version
```

Comment:
Though I think we should call this add-on 'extension',  is it intentional to use 'add-on's in this document?
If you are convinced that we should use 'add-on', then I am fine with that. :)

Reply:
Mostly because I copied it from another place :)